extensions/amp-ad-exit/0.1/config.js

```diff
@@ -42,7 +42,8 @@ export let AmpAdExitConfig;
  *   finalUrl: string,
  *   trackingUrls: (!Array<string>|undefined),
  *   vars: (VariablesDef|undefined),
- *   filters: (!Array<string>|undefined)
+ *   filters: (!Array<string>|undefined),
+ *   behaviors: (BehaviorsDef|undefined)
```

Comment:
based on the code, I think this type should be BehaviorDef and BehaviorsDef shouldn't exist. Otherwise you'd access it as target.behaviors["???"].clickTarget

Reply:
Yes that's true.  I updated so that it would just be BehaviorsDef, with no BehaviorDef.